### ISSUE TITLE
Did DORY can be used in any Net?

### ISSUE THREAD

**Issue by ChenMathilda** [pulp-platform/dory#3]
Hi,
I noticed that in the readme file, the Limitations Part mentioned:_**The DORY framework is currently tested on feed-forward networks with single-wire residual connections.**_

I want to know that can Dory be used in any Neural Networks or other Deep Learning Networks ？

**Comment by FrancescoConti** [comment]
Purely feed forward networks such as Deep NNs (VGG, MobileNet V1) and those with single-wire residuals (ResNet, MobileNet V2).
Let us know the kind of networks you are interested in.

**Comment by Zyq990127** [comment]
Hi,
Does Dory consider supporting detection networks such as Yolo?

**Comment by ChenMathilda** [comment]
> Hi, Does Dory consider supporting detection networks such as Yolo?

Hi，
I think if YOLO can meet the following requirements, it should be possible to use Dory deployment:
Pointwise Convolution (+ BatchNorm + Relu)
DepthWise Convolution (+ BatchNorm + Relu)
Max Pooling (+ BatchNorm)
Average Pooling (+ BatchNorm)
Add (+ BatchNorm + Relu) -- NOT FULLY TESTED
Linear Layer (+ BatchNorm + Relu)
Linear Layer 32 bits output -- final layer
